pom.xml

```diff
@@ -209,7 +209,7 @@
         <persistence-api.version>1.0</persistence-api.version>
         <person-directory.version>1.7.1-M1</person-directory.version>
         <pluto.version>2.1.0-M3</pluto.version>
-        <portlet-api.version>2.0</portlet-api.version>
+        <portlet-api.version>1.0</portlet-api.version>
```

Comment:
Calling it portlet-api.version is misleading since it is artifact 1.0 but NOT portlet-api 1.0.  I suggest to rename this property to portlet-2-1-api.version or in this case perhaps remove it.